### ISSUE TITLE
does not work on twitter.com

### ISSUE THREAD

**Issue by hadifarnoud** [mamal72/persian-twitter#1]
none of the fonts work
<img width="1279" alt="screen shot 1395-09-14 at 11 24 20 pm" src="https://cloud.githubusercontent.com/assets/589808/20868774/e7117896-ba78-11e6-95fc-98c4b6e0bc50.png">

**Comment by mamal72** [comment]
Hey. Thanks for opening this issue.
If you check the source of the extension, you can see sth like the following image. 

![image](https://cloud.githubusercontent.com/assets/2822264/20869219/df951f0a-ba82-11e6-9ae9-2140af4715b5.png)

Loading resources in WebExtensions are different in Chrome & Firefox. I'm using both `moz` and `chrome` prefixed URLs for loading fonts to make it work in both Chrome & Firefox.

So, having such error in the console is normal and I have no other way to fix it. It **doesn't affect the functionality** of the extension. It's just a 404 error for `moz` prefixed resources in Chrome and a 404 for `chrome` prefixed resources in Firefox developer tools console.

I'll appreciate any better workaround for this problem. 🙏

**Comment by mamal72** [comment]
You can check the source of mentioned part [here](https://github.com/mamal72/persian-twitter/blob/master/app/styles/page.css).

**Comment by hadifarnoud** [comment]
why not use a different css for each browser?

**Comment by hadifarnoud** [comment]
and the main issue for me is that it does not work in Chrome. I thought those errors were the reason but they're not

**Comment by mamal72** [comment]
Hm.. That's a good idea but it's not like that. As the content scripts are defined in `manifest` file, they can't be changed depending on a condition or sth like that. I should build two versions of the extension. I'll change the gulp tasks to achieve this. 🤔 👍 

Can you provide more info? Which OS? Which version of Chrome? Any error in the console? Can you completely quit and start your browser? 🤔

**Comment by mamal72** [comment]
Check this [release](https://github.com/mamal72/persian-twitter/releases/tag/v2.0.6). It'll be on CWS soon.
Thanks for your suggestion about separating builds. 🙏

**Comment by hadifarnoud** [comment]
looking forward to testing it. I'm using Ghost Browser which is a fork of Chrome. it uses Chrome 53.0.2785.116 engine. I'm on MacOS 10.12

**Comment by mamal72** [comment]
I didn't test it on that browser. There might be some issued or conflicts. 🤔 
If it doesn't work with the new update, please check the `class` property of `body` element. This extension is working by adding a `persian-twitter-X` class to the body element. The `X` is something like `vazir`. Classes like `persian-twitter-vazir` are defined in CSS content script and are injected in Twitter and TweetDeck pages. When you choose a font in the extension popup dialog, all `persian-twitter-X` classes are removed from body and then `persian-twitter-FONT-NAME` will be added to body element. The `FONT-NAME` is the value of the selected option. So the body font-family will be changed and then, default font is changed in that class.

![image](https://cloud.githubusercontent.com/assets/2822264/20962231/c8a66b80-bc7d-11e6-8d43-df57d95416d8.png)

### TL;DR:
Body should have `persian-twitter-vazir` class when you Choose Vazir font. You should not have any other extension or user style which overrides font-family of body or children elements of it.

**Comment by hadifarnoud** [comment]
my issue is even if I change the font, it still goes back to default font. it stays as `persian-twitter-default` regardless of what I choose.

it works just fine on latest version of Chrome btw. Just tested it with Chrome.

Checkout Ghost Browser. You'll love it

**Comment by mamal72** [comment]
Glad to hear that. 👍 

And about the Ghost browser... that's strange. It might be a bug with another extension or some internal extension or some changes in that fork. I may test it and find out what's the problem. Thanks for reporting. 🙏 

I think we can close this issue. Feel free to do it. 😁👍

**Comment by CoolSciGuy** [comment]
Hi folks,

Since twitter updated to a new scheme last June, the app stopped working on Chrome+windows 7. It works on Chrome+mac.

![Capture](https://user-images.githubusercontent.com/18423462/62707944-55737480-b9c0-11e9-8ec6-9c93f5189695.JPG)

**Comment by mamal72** [comment]
I'm gonna put some time on it and update it to support new migrated Twitter mobile. I'll release an update in a week to fix the issues with new Twitter.
